charts/merlin/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: v0.38.0-rc1
+appVersion: v0.39.1
```

Comment:
This technically doesn't install a version of Merlin that exposes the tolerations and node selectors since that feature has not been released with a new Merlin tag yet (v0.39.2? or should I just cut a new RC tag and update this value here?).

Reply:
oh that make sense, the latter then, lets hold this for a while

Reply:
I've just updated this to the latest version v0.40.0.